definitions/checks/disk/available_space.rb

```diff
@@ -0,0 +1,23 @@
+module Checks
+  module Disk
+    class AvailableSpace < ForemanMaintain::Check
+      metadata do
+        label :available_space
+        description 'Check if root(/) partition has enough space to proceed with upgrade'
+        tags :default
+      end
+
+      MIN_SPACE_IN_MB = 4096
+
+      def run
+        assert(!enough_space?, "System has less than #{MIN_SPACE_IN_MB / 1024}GB space available"\
+              ' on root(/) partition')
+      end
+
+      def enough_space?
+        io_obj = ForemanMaintain::Utils::Disk::IODevice.new('/')
+        return true if io_obj.available_space < MIN_SPACE_IN_MB
```

Comment:
`?` mark at the end of method name indicates that a method returns a `boolean` value.
Please modify method so that it returns only boolean value.